sop/modules/OSD/pages/CephNodeDown.adoc

```diff
@@ -76,9 +76,6 @@ include::partial$deploy_toolbox.adoc[]
 ceph status
 ----
 
-How long recovery takes will depend on the size of the Ceph cluster.
-WIP: Note about health status and no recovery for small quick clusters.
```

Comment:
there's a WIP - on link to node replacement SOP in this doc, which would need to be changed too